module-cache.ts

```diff
@@ -0,0 +1,6 @@
+import {Browser} from './browser';
+export class ModuleCache {
+    constructor(browser: Browser) { var dictionary = {}; }
+    get(names: string[]) => dictionary[names];
```

Comment:
Not all of the names will be in the cache. Change declaration to be `get(...): void { ... }`. Message me and we'll talk about how to fill in the body of this method.